server/utils/applications/getApplicationData.ts

```diff
@@ -79,6 +81,14 @@ const firstClassFields = <T>(
     caseManagerIsNotApplicant,
     noticeType,
   } as FirstClassFields<T>
+
+  const startDateOutsideOfTimescales = startDateOutsideOfNationalStandardsTimescales(application)
+  if (startDateOutsideOfTimescales) {
+    const reasonForShortNotice = retrieveQuestionResponse(application, ReasonForShortNotice, 'reason')
```

Comment:
I think it would be better if we followed the same pattern as all the other properties in the object that is returned from this function.  
We should create a util function that returns `reasonForShortNotice` and `reasonForShortNoticeOther` and add this to the return value. `applicationAndCaseManagerDetails` is a good example of this.